openstack/hermes/templates/etc/_logstash.conf.tpl

```diff
@@ -1,13 +1,12 @@
 input {
 
 {{ range $key, $value := .Values.hermes.rabbitmq.targets }}
```

Comment:
To me $key and $value are not explanatory. Can they be renamed? I've tried to understand what $key is actually holding but there is also no example in the [values.yaml](https://github.com/sapcc/helm-charts/blob/master/openstack/hermes/values.yaml#L23-L28)

Checking in the config repo it is the replica. So maybe this would be a better naming here:

```suggestion
{{ range $replica_name, $replica := .Values.hermes.rabbitmq.targets }}
```

Then the bottom part also would read easier:

```
    user => "${RABBITMQ_{{ upper $replica_name }}_USER}"
    password => "${RABBITMQ_{{ upper $replica_name }}_PASSWORD}"
```